frontend/src/api.js

```diff
@@ -101,9 +102,11 @@ function query(sql, signal) {
 }
 
 function queryExport(sql) {
-  const url = new URL(apiUrl('/export'));
-  url.searchParams.append('query', sql);
-  return url.toString();
+  const rawUrl = apiUrl('/export');
+  const href = new URL(window.location.href);
```

Comment:
Sorry, I meant to include this in the previous review.

Instead of using `window.location` and then discard it, please check if we can do this:
```js
const params = new URLSearchParams();
params.append('query', sql);

return `${rawUrl}?${params.toString()}`;
```

Reply:
It works! And it's much more cleaner this way 👍

Reply:
Now I see in travis the tests fail because `URLSearchParams` is not available in node :thinking:
Maybe you can find a polyfill lib and initialize it for the node tests in `setupTests.js`

Reply:
Yup I saw that jest doesn't not support `URLSearchParams` until version 22, and that version comes with `react-scripts>=2.0.0`.